picki/app/events/page.js

```diff
@@ -3,16 +3,20 @@
 import { useState } from 'react';
 import ArrowButton from '../components/arrowButton/ArrowButton';
```

Comment:
When you are importing, be careful of using relative paths. This is usually fine for static html/js/css sites, but any app with a build/run process rearranges your files when it get built.

Instead use the @ import alias. 

`import ArrowButton from '@...'`